src/util/config.cpp

```diff
@@ -1280,38 +1308,40 @@ irep_idt configt::this_architecture()
     #else
     this_arch="powerpc";
     #endif
-  #elif defined(__sparc__)
-    #ifdef __arch64__
-    this_arch="sparc64";
-    #else
-    this_arch="sparc";
-    #endif
-  #elif defined(__ia64__)
+#elif defined(__riscv)
+  this_arch = "riscv64";
+#elif defined(__sparc__)
+#ifdef __arch64__
```

Comment:
Suggest to keep the useful indenting of these lines?

Reply:
That would require disabling clang-format.

Reply:
go on, type `// clang-format off`. Live dangerously!